models/pattern/core/pattern.go

```diff
@@ -42,6 +42,7 @@ type Service struct {
 	Name        string            `yaml:"name,omitempty" json:"name,omitempty"`
 	Type        string            `yaml:"type,omitempty" json:"type,omitempty"`
 	Namespace   string            `yaml:"namespace" json:"namespace"`
+	Version     string            `yaml:"version" json:"version"`
```

Comment:
@Abhishek-kumar09 is this the pattern version or other? How can we make this more clear?

Reply:
This is a Service Struct, thus it defines the service (Components in Extensions) version.